Recurly.Tests/PagerTest.cs

```diff
@@ -50,62 +48,46 @@ public void EnumerableTest()
             pager.Dispose();
```

Comment:
Is this something we're going to build out later or is it something that's in the Interface but doesn't make sense for our context?

Reply:
This method must be present and callable to conform to the IEnumerable interface. I don't think it needs to do anything in our case though because we do not hold any resources.